bin/pygrb/pycbc_pygrb_minifollowups

```diff
@@ -94,11 +87,12 @@ def make_timeseries_plot(workflow, trig_file, snr_type, central_time,
     node.add_opt('--x-lims='+x_lims)
     # Plot title
     if ifo is not None:
-        title_str = "'%s SNR at %.3f (s)'" %(ifo, central_time)
-        node.add_opt('--central-time', central_time)
+        title_str = "'%s SNR at %.3f (s)'" % (ifo, central_time)
+        node.add_opt('--trigger-time', central_time)
     else:
-        title_str = "'%s SNR at %.3f (s)'" %(snr_type.capitalize(), central_time)
-        node.add_opt('--central-time', central_time)
+        title_str = "'%s SNR at %.3f (s)'" % (snr_type.capitalize(),
+                                              central_time)
+        node.add_opt('--trigger-time', central_time)
```

Comment:
You can take `node.add_opt('--trigger-time', central_time)` out of the if/else statement.